### ISSUE TITLE
Add a new pop up screen which asks for confirmation before importing a certificate

### ISSUE THREAD

**Issue by Ein-Tim** [Digitaler-Impfnachweis/covpass-ios#71]
## Current Implementation
Scanning an EU digital COVID certificate directly imports it, without asking the user whether the intention is really to import it or to verify if. 

## Suggested Enhancement 
The following is a concrete suggestion for solving https://github.com/Digitaler-Impfnachweis/covpass-android/issues/103.

When a user scans a certificate with the app, a new pop up screen should be shown, where the user has to confirm that the intuition really is to import a certificate and not to verify it.
As an example, I suggest to take a look at how it's implemented "AntiCovid":

<img src="https://user-images.githubusercontent.com/67682506/143941316-a2bf7025-baa2-4e59-a737-791eccf52f25.PNG" width="300">

## Expected Benefits
This change would lower / nearly mitigate that the CovPass-App is used to "verify" DCCs from other people. 

## Related issues

- #67
- https://github.com/Digitaler-Impfnachweis/covpass-android/issues/103
- https://github.com/Digitaler-Impfnachweis/covpass-android/issues/99

**Comment by alexcimander** [comment]
hey everyone!

we are currently working on multiple features that tackle the misuse of the CovPass as a verifier-app. A screen asking if the import is actually wanted is one of them. We are planning to release it with the next release. We will keep you updated about it! Thank you for the great ideas 👍 🎆

**Comment by moabits** [comment]
Possible other solution approach: https://github.com/corona-warn-app/cwa-wishlist/issues/601

**Comment by Ein-Tim** [comment]
I consider this as implemented with version 1.15. See: https://github.com/Digitaler-Impfnachweis/covpass-android/issues/103#issuecomment-1013392983

**Comment by Ein-Tim** [comment]
Reopening for reason given in: https://github.com/Digitaler-Impfnachweis/covpass-android/issues/103#issuecomment-1014529356

**Comment by Ein-Tim** [comment]
The screen is not shown always but only if there are already 10 persons registered in the app.

**Comment by timokoenig** [comment]
Yep, not a good implementation. This should be shown for every new person that is being added to the app.

**Comment by Ein-Tim** [comment]
I also don't hate how CovPass handles it but when it comes to misuse prevention (which is what we are discussing here), I definitely think CWAs solution is more effective.

**Comment by ingowerren** [comment]
In 2021, the European Union introduced the digital COVID certificate as an EU-wide recognised proof of vaccination, recovery or negative testing for COVID-19. This regulation expired on 30 June 2023. On a transitional basis, the EU will continue to operate the technical systems to enable cross-border certificate checks until 31 December 2023. The transitional operation will end on that date.
In Germany, the possibility of issuing digital COVID certificates such as vaccination and recovery certificates will therefore be discontinued on 31 December 2023. The CovPass app and the CovPassCheck app will therefore also be discontinued. The CovPass app will be switched to wallet mode, where the stored certificates will be retained.

What does this mean for you?
All functionalities will continue to beavailable until 31 December 2023.

**From 1 January 2024:**

Support for the app will no longer be offered.

- The apps will no longer be available for (re)download in the app store.
- No new functions will be published.
- Further information can also be found in the CovPass app privacy policy.
